src/Hyperwallet/Hyperwallet.php

```diff
@@ -1803,4 +1826,59 @@ private function updateTransferMethod($userToken, $transferMethod, $transferMeth
                 ), $transferMethod, array());
     }
 
+    /**
+     * Create a bank card status transition
+     *
+     * @param string $userToken The user token
+     * @param UserStatusTransition $transition The status transition
+     * @return UserStatusTransition
+     *
+     * @throws HyperwalletArgumentException
+     * @throws HyperwalletApiException
+     */
+
+    public function updateVerificationStatus($userToken,$verificationStatus){
+        if (empty($userToken)) {
+            throw new HyperwalletArgumentException('userToken is required!');
+        }
+        if (empty($verificationStatus)) {
+            throw new HyperwalletArgumentException('verification status is required!');
+        }
+        if($verificationStatus != User::VERIFICATION_STATUS_REQUESTED){
+            throw new HyperwalletArgumentException("Expected verification status is REQUESTED!");
+        }
+        $user = $this->getUser($userToken);
+        $fromValidationStatus = $user->getVerificationStatus();
+        $expectedFromValues = array(User::VERIFICATION_STATUS_REQUIRED,User::VERIFICATION_STATUS_NOT_REQUIRED);
+        if(!in_array($fromValidationStatus,$expectedFromValues)){
+            throw new HyperwalletException("The from verification status is expected to be 'Required' or 'Not required'");
+        }
+        $user = new User(array('verificationStatus'=> User::VERIFICATION_STATUS_REQUESTED));
```

Comment:
Pls check whether we need to create the user as an array here.

Reply:
User is not an object here. It takes in array for the request body in the constructer. method signature for doPost here is (string, array, object, array). Here the object is user.